package.json

```diff
@@ -18,7 +18,7 @@
     "c": "docker-compose run --rm server-composer",
     "deploy:images": "lerna run --stream deploy:image",
     "predeploy:packages": "lerna run --stream build --scope=@serlo/{client,static-assets} --include-filtered-dependencies",
-    "deploy:packages": "lerna run --stream deploy:package",
+    "deploy:packages": "cross-env DEPLOY=true lerna run --stream deploy:package",
```

Comment:
Please don't, this is intentionally hard to deploy manually.

Reply:
It is still hard now. Due to https://github.com/serlo/serlo.org/blob/e52cdf34ef3ec3007aa4804f4f8b0e8692cdf5a2/packages/private/cloudflare/index.ts#L109-L111 one gets an error when `CF_KEY` is not set. So nothing happens when one accidentally executes `yarn deploy:packages`. So only difference is that now the error message is more meaningful (I always wondered why the script said to me that it skips the deployment until I remembered that I have to set env variables...). Also with something like `CF_KEY=foo yarn deploy:packages` one gets the error `@serlo/static-assets: ✖  fatal     Error: CF: error getting "serlo-org-static-assets@2.0.0": {"success":false,"errors":[{"code":10000,"message":"Authentication error"}]} ` so without a valid token nobody can execute the script and we are the only ones who can create tokens for cloudflare...

Reply:
With https://github.com/serlo/serlo.org/blob/e52cdf34ef3ec3007aa4804f4f8b0e8692cdf5a2/packages/private/cloudflare/index.ts#L109-L111 it is okay to delete the necessity of `DEPLOY=true`. It seems that it is just a safety against falsely executing `yarn deploy:images`.

Reply:
IMHO: either remove `DEPLOY` completely or don't set it by default.

Reply:
Removed `DEPLOY` in https://github.com/serlo/serlo.org/pull/666/commits/728cf3c93d840cec50f0af2c3a08462de3e97fa2